bin/sl-run.js

```diff
@@ -17,6 +17,14 @@ var agentOptions = {
   logger: config.logger,
 };
 
+var tracer = require('concurix');
+var tracerOptions = {
+  accountKey: config.clustered.toString()+process.pid.toString(),
+  archiveInterval: 60000,
+  api_host: 'localhost', // XXX(seto) get it from Hotname parameter in Deploy
```

Comment:
Need to replace HTTP with node IPC. We'll talk more, but see lib/express-records.js for example.